utils/oscap.8

```diff
@@ -75,6 +75,11 @@ You may specify OVAL Definition files as the last parameter, XCCDF evaluation wi
 Select a particular profile from XCCDF document.
 .RE
 .TP
+\fB\-\-rule RULE\fR
+.RS
+Select a particular rule from XCCDF document. Only this rule will be evaluated.
```

Comment:
Same as in the --help text:
Can you specify that profile is necessary to be specified for this option and what are the reasons?